Package.swift

```diff
@@ -10,6 +10,13 @@ let package = Package(
         .library(name: "Q42Stats", targets: ["Q42Stats"])
     ],
     targets: [
-        .target(name: "Q42Stats", dependencies: [])
+        .target(
+            name: "Q42Stats",
+            dependencies: [],
+            path: "Sources",
+            resources: [
+                .copy("PrivacyInfo.xcprivacy"),
```

Comment:
The AppAuth library has recently [added a privacy manifest](https://github.com/openid/AppAuth-iOS/blob/master/Package.swift), and they did it in the exact same way. So LGTM!